packages/web/src/pages/schedule.tsx

```diff
@@ -44,7 +44,7 @@ const Schedule: NextPage = () => {
   }, [toast]);
 
   return (
-    <PageContainer pageTitle={'Schedule'} heading={'Events'}>
+    <PageContainer pageTitle={'Schedule'} heading={'Schedule'}>
```

Comment:
oopsies my bad :)

Reply:
This wasn't you and I originally saw it during code review but thought I'd leave it; it later started to bug me so I changed it